forge/routes/ui/index.js

```diff
@@ -48,9 +74,12 @@ module.exports = async function (app) {
             reply.redirect('/setup')
             return
         }
+        if (app.config.telemetry.frontend?.plausible) {
+            app.log.warn('Configuration found for Plausible. Please note that support for Plausible will be deprecated after FlowForge 0.9')
+        }
         // check if we need to inject plausible
-        if (app.config.telemetry.frontend?.plausible?.domain) {
-            const injectedContent = await injectPlausible(app.config.telemetry.frontend.plausible.domain, app.config.telemetry.frontend.plausible.extension)
+        if (app.config.telemetry.frontend) {
+            const injectedContent = await injnectAnalytics(app.config.telemetry.frontend) // injectPlausible(app.config.telemetry.frontend.plausible.domain, app.config.telemetry.frontend.plausible.extension)
```

Comment:
Still has typo in name
```suggestion
            const injectedContent = await injectAnalytics(app.config.telemetry.frontend) // injectPlausible(app.config.telemetry.frontend.plausible.domain, app.config.telemetry.frontend.plausible.extension)
```

Reply:
These had just been addressed - think you;'re viewing outdated code?